python/ray/autoscaler/cli_logger.py

```diff
@@ -513,6 +513,30 @@ def confirm(self, yes, msg, *args, _abort=False, _default=False, **kwargs):
 
         return res
 
+    def prompt(self, msg: str, *args, **kwargs):
+        """Display a confirmation dialog.
```

Comment:
the doc string should probably be changed

Reply:
fixed, thanks